scripts/mkcloud

```diff
@@ -1237,6 +1247,11 @@ function sanity_checks()
     if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
+
+    if [[ " ${steplist[*]} " == *" batch "* ]] && [[ "$scenario" == '' ]] ; then
```

Comment:
Sure, in the whitespace case the quoting makes sense.  And I understood why you did it ;-)

> Is there a better way to check if an array contains an element?

I don't think so:

- http://stackoverflow.com/questions/3685970/check-if-an-array-contains-a-value
- http://stackoverflow.com/questions/14366390/bash-if-condition-check-if-element-is-present-in-array

Another example of why #shellsucks ...